### DISCUSSION TITLE
Subscribed to github copilot but can't use it in intellij

### DISCUSSION THREAD

**Discussion by Felistas** [community/community#30618] [Copilot Conversations]
- I have installed and uninstalled the github co pilot a couple of times, logged in and activated my device but its still not working. Anyone faced this issue before?

Both intellij and vscode nothing works

**Comment by davecheney** [comment]
@Felistas i'm sorry you are having difficulties. Looking at your telemetry it looks like you are making and receiving completions. Can you please tell me a bit more about what you tried, and what happened. Thank you.

**Reply by Felistas** [reply]
Had to update the plugin but it works now
